WordPress/Classes/ViewRelated/System/Notices/NoticeView.swift

```diff
@@ -239,7 +239,11 @@ class NoticeView: UIView {
         cancelBackgroundView.addTrailingBorder()
 
         actionButton.titleLabel?.adjustsFontForContentSizeCategory = true
-        actionButton.setTitleColor(.white, for: .normal)
+        if #available(iOS 13.0, *) {
+            actionButton.setTitleColor(.label, for: .normal)
```

Comment:
Could you move this to the NoticeStyle extension as another var? `actionButtonTitleColor` or something, and handle the `if #available` there?

Reply:
Sure!